File: FizzBuzz/FizzBuzz.java
import java.util.ArrayList;
import java.util.List;

/**
 * Created by XiaochengWen on 16/10/15.
 */
public class FizzBuzz {
    public List<String> fizzBuzz(int n) {
        ArrayList<String> out = new ArrayList<String>();
        for (Integer i=1;i<=n;i++){
            String a = "";
            if (i%3==0){
                a += "Fizz";
            }
            if (i%5==0){
                a += "Buzz";
            }
            if (a==""){
                a = i.toString();
            }
            out.add(a);
        }
        return out;
    }
}

File: FizzBuzz/FizzBuzz.py
__author__ = 'XiaochengWen'
class Solution(object):
    def fizzBuzz(self, n):
        """
        :type n: int
        :rtype: List[str]
        """
        out = []
        for i in range(1,n+1):
            a = ""
            if (i%3==0):
                a += "Fizz"
            if (i%5==0):
                a += "Buzz"
            if (not a):
                a = str(i)
            out.append(a)
        return out